src/bitmap/immutable.rs

```diff
@@ -7,6 +7,7 @@ use super::{
     utils::{count_zeros, fmt, get_bit, get_bit_unchecked, BitChunk, BitChunks, BitmapIter},
     MutableBitmap,
 };
+use num_traits::zero;
```

Comment:
is this needed? I can't find where we use it.

Reply:
Hmm.. Don't think so as well. :thinking: